pkg/controller/daemon/daemon_controller.go

```diff
@@ -706,6 +704,7 @@ func (dsc *DaemonSetsController) updateNode(logger klog.Logger, old, cur interfa
 	}
 	// TODO: it'd be nice to pass a hint with these enqueues, so that each ds would only examine the added node (unless it has other work to do, too).
 	for _, ds := range dsList {
+		// If NodeShouldRunDaemonPod needs to use other properties of Node, it needs to update shouldIgnoreNodeUpdate.
```

Comment:
Let's clarify, by saing 

... needs to uses other than Labels and Taints (mutable) properties of node, ...